### PR TITLE
Add adityatheoctocatdev to the list

### PR DESCRIPTION
**Pull request by adityatheoctocatdev**

### CODE REVIEW

File: public/data/adityatheoctocatdev.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Aditya Patel",
+  "bio": "DevRel at @thesemicolondevs 🥑·",
+  "avatar": "https://github.com/adityatheoctocatdev.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "http://github.com/adityatheoctocatdev",
```
Comment (rohan-kulkarni-25) [inline public/data/adityatheoctocatdev.json:8]: - Please make `https` to  Github URL

File: public/data/adityatheoctocatdev.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "http://twitter.com/adptheoctcatdev",
```
Comment (rohan-kulkarni-25) [inline public/data/adityatheoctocatdev.json:13]: - Please make `https` to twitter URL

**Review by rohan-kulkarni-25 — CHANGES_REQUESTED** [review]

File: public/data/adityatheoctocatdev.json (hunk 1 above)
Comment (Panquesito7) [inline public/data/adityatheoctocatdev.json:8]:
```suggestion
      "url": "https://github.com/adityatheoctocatdev",
```

File: public/data/adityatheoctocatdev.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Aditya Patel",
+  "bio": "DevRel at @thesemicolondevs 🥑·",
```
Comment (Panquesito7) [inline public/data/adityatheoctocatdev.json:3]:
```suggestion
  "bio": "DevRel at @thesemicolondevs 🥑",
```

File: public/data/adityatheoctocatdev.json (hunk 2 above)
Comment (Panquesito7) [inline public/data/adityatheoctocatdev.json:13]:
```suggestion
      "url": "https://twitter.com/adptheoctcatdev",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by niteshseram — APPROVED** [review]
LGTM

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍 
Thanks for reviewing this pull request, folks! 🙂

**Pull request merged**